packages/docregistry/src/registry.ts

```diff
@@ -1191,21 +1191,6 @@ namespace DocumentRegistry {
       iconClass: 'jp-MaterialIcon jp-ImageIcon',
       fileFormat: 'base64'
     },
-    {
```

Comment:
Why remove these? I thought that somebody wanted the `.raw` extension specifically at some point.

Reply:
What browser supports an `img` with a raw extension? I don't think a `.raw` file format even is formally specified, let alone supported.

Reply:
Do you know who asked for it or what context it was in?

Reply:
Unfortunately, I don't remember, and a quick look through the issues did not come up with anything.